machiavelli-cards/src/main/java/pl/jellytech/machiavelli/cards/dtos/BaseResponseWrapper.java

```diff
@@ -0,0 +1,15 @@
+package pl.jellytech.machiavelli.cards.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
just for info - Lombok also has @Data which has RequiredArgsConstructor, getter, setter, toString and some other shit. 
Personally I prefer this way (having the annotations separately - more readable), but just so you know :)